resources/config/samples.txt

```diff
@@ -0,0 +1,3 @@
+Sample
```

Comment:
Sorry, I just don't understand the reasoning behind having these as two files. You've basically just made the "units" file what "samples.txt" used to be. Why wouldn't we just have all of this information in the same file?

Reply:
This is looking towards downstream use. Say, for example, you want to process some samples against a different database, or include a set of metadata that you will access for a visualization step. Including metadata files at the appropriate level of resolution facilitates that.

Reply:
Ah, so `samples.txt` is just the metadata file. I definitely agree that we should include it in the beginning, and I guess it would make it cleaner to have the file paths in a different file. 

Will this rule need to check that the `Sample_IDs` column in both files are the same?

I also wonder if there is any sort of formatting that we could enforce on the `samples.txt` file to facilitate uploading these samples to a database like SRA, EBI, or QIITA.

Reply:
Yeah, there absolutely is! We can include a metadata file schema for that, and specify required or optional values. 

It is good to add some logic to validate the sample id representation. I prefer to have it so that `Sample` in `samples.txt` must be found in `units.txt` but not vice versa, so that you can for example comment out a bunch of samples from `samples.txt` if you want to test a subset.

Reply:
That sounds like a great idea!

So then `units` would act as a master list (maybe even include paths to all of the samples in the entire lab?), and the `samples` would specify which samples we specifically want to run, and include all of their associated metadata.

Reply:
That could definitely be possible. Although in practice I'd guess it would be a little cumbersome to keep it as a flat file -- I love the idea of dynamically generating it from a db though. :D